app/src/main/java/org/simple/clinic/widgets/Views.kt

```diff
@@ -300,6 +301,12 @@ fun ViewPropertyAnimator.setDuration(duration: Duration): ViewPropertyAnimator {
   return setDuration(duration.toMillis())
 }
 
+fun RadioGroup.check(@IdRes id: Int, onCheckedChangeListener: RadioGroup.OnCheckedChangeListener) {
```

Comment:
Same comment as earlier regarding the naming the extension method that manages a `TextWatcher`.